packages/web/src/dev/ValidatorsList.scss

```diff
@@ -0,0 +1,233 @@
+// TODO: Move to files
```

Comment:
Agree with @aaronmgdr - just for consistency's sake here, I would prefer not to introduce another method of using CSS. We use react-native-web's stylesheet everywhere else, and try to stick to flexbox (https://flexboxfroggy.com/) as much as possible.

Reply:
The problem is that is not possible to align the content as designed. From my side, I'm ok changing it to the table component, but it doesn't look as expected.
@cmcewen do you think that is ok?

Reply:
how does it look? 
what the alignment that doesn't seem to work?

Reply:
![image](https://user-images.githubusercontent.com/1885747/73198554-762cb380-4133-11ea-980e-383e62ffb845.png)
I mean that. The solution is hardcode the width of each column. On this case, I suppose that is not a problem, but the table won't be fluid anymore.

Reply:
ok.  @cmcewen whats your thought?

Reply:
Since this is obviously a table, one option would be to use react-native-web's `createElement` method like we do for video tag on the home/about page. this way you get a native table element which you can style with `Stylesheet.create`